src/openlifu/db/session.py

```diff
@@ -72,9 +92,8 @@ class Session:
     """Approval state of virtual fit. `None` if there is no approval, otherwise this is the ID
     of the target for which virtual fitting has been marked approved."""
 
-    transducer_tracking_approved: Optional[bool] = False
-    """Approval state of transducer tracking. `True` means the user has provided some kind of
-    confirmation that the transducer transform in this session agrees with reality."""
+    transducer_tracking_results: Optional[List[TransducerTrackingResult]] = field(default_factory=list)
+    """List of any transducer tracking results"""
```

Comment:
It can just be a List instead of an Optional[List]

(the case of an empty list is meaningful, and I can't think of anything else that `None` could mean)